src/components/Article/SectionTitle/index.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import Block from '../../Block'
+import Typography from '../../Typography'
+
+const SectionTitle = ({
+  children,
+  color,
+  element,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  lineHeight,
+  maxWidth,
+  mb,
+  value
+}) => {
+  return (
+    <Block maxWidth={maxWidth} width='100%'>
+      <Typography 
+        color={color}
+        dangerouslySetInnerHTML={value}
+        element={element}
+        fontFamily={fontFamily}
+        fontSize={fontSize[0]}
+        fontWeight={fontWeight}
+        lineHeight={lineHeight[0]}
+        lg={{
+          fontSize: fontSize[1],
+          lineHeight: lineHeight[1],
+          mb: mb[1]
+        }}
+        mb={mb[0]}
+        mt={3}>
+        {children} 
+      </Typography>
+    </Block>
+  )
+}
+
+SectionTitle.defaultProps = {
+  color: 'neutral2',
+  element: 'h2',
+  fontFamily: 'primary',
+  fontSize: ['20px', '20px'],
+  fontWeight: 700,
+  lineHeight: ['120%', '120%'],
+  mb: ['2', '2'],
```

Comment:
Esses valores aqui não tem que ser number?

Lembro-me que a função de margin pode receber dois tipos...
Caso receba `string` ela coloca a string direto como valor da propriedade margin no css, sem fazer nenhum tratamento ou cálculo.
`margin-bottom: ${prop};`

Caso receba number ela espera a realização do calc com a unidade e adiciona px no final:
`margin-bottom: ${prop * factor}px;`

O problema de passarmos string é que no CSS pode ser que fique da seguinte forma:
`margin: 2;` ao invés de `margin: 16px;`

Pode dar um check nisso pra nós?

Reply:
Cara a gente sempre fez dessa maneira. Todos os margins nos componentes do Article são chamados dessa maneira. Se for corrigir aí, teria que mudar todos.

Reply:
Chegou a verificar se no css final ficou tudo ok?

Reply:
Ajustado, vou aplicar para o restante dos componentes.